inql/actions/sendto.py

```diff
@@ -30,6 +30,8 @@
 from inql.actions.browser import URLOpener
 from inql.utils import make_http_handler, HTTPRequest
 
+LISTENING_PORT = 0xD09e115ec % (2 ** 16)
```

Comment:
The reason for having a random port was to not crash with multiple instances of InQL. Can we have an incremental port try if the port is occupied and still have multiple instances of InQL?

Reply:
Great idea! I implemented the graceful fallback.